src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,67 @@
+import classNames from 'classnames';
+import { useContext } from 'react';
+import { deleteTodo } from '../../api/todos';
+import { Error } from '../../types/Error';
+import { AppContext } from '../AppContext/AppContext';
+import { Filters } from '../Filters';
+
+export const Footer = () => {
+  const todoData = useContext(AppContext);
+
+  const checkCompletedTodos = () => {
+    if (todoData?.todos) {
+      const { todos } = todoData;
+
+      for (let i = 0; i < todos.length; i += 1) {
+        if (todos[i].completed) {
+          return true;
+        }
+      }
+    }
+
+    return false;
+  };
+
+  const removeAllCompletedTodos = () => {
+    if (todoData?.todos) {
+      const { todos, setIsError, setTodos } = todoData;
+
+      for (let i = 0; i < todos.length; i += 1) {
+        if (todos[i].completed) {
+          deleteTodo(todos[i].id)
+            .then(() => setTodos(todos.filter(item => !item.completed)))
+            .catch(() => setIsError(Error.Delete));
+        }
+      }
```

Comment:
use map & Promise.All for this
```suggestion
      for (let i = 0; i < todos.length; i += 1) {
        if (todos[i].completed) {
          deleteTodo(todos[i].id)
            .then(() => setTodos(todos.filter(item => !item.completed)))
            .catch(() => setIsError(Error.Delete));
        }
      }
```